### DISCUSSION TITLE
verbose option for fit model

### DISCUSSION THREAD

**Discussion by FABoulanger** [ourownstory/neural_prophet#237] [Q&A - get help using NeuralProphet] [answered]
Hello,

in the fit function I don't see a verbose option to remove the epoch progress bar. Do you know of another way? thank you in advance

**Comment by HansikaPH** [comment, marked as answer]
The fit function has the option use_tqdm=False which you can use to do this. The name of the option might be a bit confusing but that can basically let you get rid of the progress bar. This is set to True by default.

**Comment by ourownstory** [comment]
@FABoulanger in addition, you can set the general verbosity of all outputs to a certain level of your choosing by setting
```
from neuralprophet import set_log_level
set_log_level("ERROR")
```

**Comment by DeastinY** [comment]
Both of those don't work for me. There's still progress bars being plotted :/ Even manually removing tqdm from the source does not seem to help. I guess there's an external dependency somewhere?

**Reply by ourownstory** [reply]
@DeastinY Yes, the problem lies with https://github.com/davidtvs/pytorch-lr-finder.
By setting a manual lr, you should be able to get rid of it.
We might have to fork the above repo and strip it of all that is meant for in-notebook single run usage.
